crates/downloader/src/main.rs

```diff
@@ -371,15 +371,15 @@ async fn main() -> anyhow::Result<()> {
         tasks.spawn(download_and_extract_from_gh(
             c_api,
             Stripping::FirstDir,
-            &output,
+            output.join("c_api"),
```

Comment:
本当に`DownloadTarget`に合わせるのなら、そっちの名前に合わせてkebab-caseにする、という考えかたもあるかも…？？

```suggestion
            output.join("c-api"),
```

Reply:
これはファイル・ディレクトリ名のケースは統一の方が分かりやすいと思います！
アンダースコアで良さそう。